test/runtests.jl

```diff
@@ -856,13 +856,13 @@ end
                 """)
 
                 testaln("""
-                 ACGT  
```

Comment:
I think these are what are causing the tests failiure, there needs to be two spaces following the last letter, which some people's editors remove.